### ISSUE TITLE
FOUC for outlined Select with pre-selected value

### ISSUE THREAD

**Issue by lamtranweb** [jamesmfriedman/rmwc#283]
Using RMWC 1.8.3.

If `Select` component has a pre-selected value, FOUC occurs briefly until the `Select` updates with the correct style.

This is because `<label class="mdc-floating-label">` is missing `mdc-floating-label--float-above` class and `<div class="mdc-notched-outline"><svg>...</svg></div>` is missing `mdc-notched-outline--notched` and `mdc-noteched-outline`'s inner SVG is not showing the notched border.

In short, `Select` with pre-selected value needs to show the selected state without the brief FOUC.

MDC discussed this here: https://material.io/develop/web/components/input-controls/select-menus/ (scroll to "Select with pre-selected option" section).

**Comment by jamesmfriedman** [comment]
Thanks for reporting, I'll look into this.

**Comment by robcaldecott** [comment]
If it helps we see the same issue when switching from nothing selected to a selected value.

Here is an example: https://codesandbox.io/s/vj950j03z5

**Comment by robcaldecott** [comment]
Here's a workaround:

https://codesandbox.io/s/2xm7jj86on

(it calls `select.foundation_.floatLabelWithValue_` during `componentDidUpdate`.

**Comment by jamesmfriedman** [comment]
Thanks for the code example and the workaround. Will verify to make sure this is fixed in #291
